swagger1/src/main/java/com/cerner/beadledom/swagger/SwaggerUiResource.java

```diff
@@ -7,13 +7,16 @@
 import com.google.common.collect.ImmutableSet;
 import java.io.InputStream;
 import java.util.Set;
+import java.util.regex.Pattern;
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.Context;
+import javax.ws.rs.core.HttpHeaders;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
```

Comment:
Are these imports unused? I don't see any other changes in this file.

Reply:
They are.. I'm not sure why I didn't get a checkstyle issue when I ran `mvn clean install`. Removed here: https://github.com/cerner/beadledom/pull/149/commits/61ab77af9cf42368041d0268f9d488dc890d3d48

Reply:
They are.. I'm not sure why I didn't get a checkstyle issue when I ran mvn clean install. Removed here: 61ab77a